Header header
int32 armor_type
float32[] point_data
